app/views/mno/extra_mobile_data_requests/index.html.erb

```diff
@@ -17,64 +17,32 @@
   <%= govuk_button_link_to 'Download requests as CSV', sort: params[:sort], dir: params[:dir], format: :csv %>
   <p class="govuk-body"><%= govuk_link_to t('page_titles.extra_mobile_data_requests_csv_update'), new_mno_extra_mobile_data_requests_csv_update_path %></p>
```

Comment:
Can we make a small tweak here so it looks like this:

![Screen Shot 2021-01-25 at 14 35 10](https://user-images.githubusercontent.com/319055/105719873-b24b4d80-5f1a-11eb-8023-38407ff25ab5.png)



See this commit for reference: https://github.com/DFE-Digital/get-help-with-tech-prototype/commit/1a850a8b3097354302c61731fba9bbe3312f3707

Classes on `app-card`:
```app-card govuk-!-margin-bottom-6 govuk-!-margin-top-4```

Reply:
Spacing changed